js/client-battle.js

```diff
@@ -1242,13 +1244,15 @@
 			buf += '<p><label class="optlabel"><input type="checkbox" name="ignoreopp"' + (this.battle.ignoreOpponent ? ' checked' : '') + '/> Ignore Opponent</label></p>';
 			buf += '<p><strong>All battles</strong></p>';
 			buf += '<p><label class="optlabel"><input type="checkbox" name="ignorenicks"' + (Tools.prefs('ignorenicks') ? ' checked' : '') + ' /> Ignore nicknames</label></p>';
+			buf += '<p><label class="optlabel"><input type="checkbox" name="righthandbattles"' + (Tools.prefs('righthandbattles') ? ' checked' : '') + ' /> Open new battles on the right side</label></p>';
```

Comment:
Another issue: whether there is enough room for a battle on the right depends on what room the left panel is showing at the time of a new battle starting. Which setup should it use to check? Home screen + battle?